python/adaptivity.py.in

```diff
@@ -333,7 +333,65 @@ def detect_colinearity(mesh, tol = 30):
 
     return bfaces_func
 
-
+def adapt_boundary_regions(original_mesh, adapted_mesh, original_regions):
```

Comment:
Ok, I will open the issue.

For the moment being I have added a few lines of code that fix this issue as a postprocessing step in the python interface. I will also temporarily leave the adapt_boundary_regions routine as it is able to find the correct IDs. As soon as the issue is resolved I will remove these routines from adaptivity.py.in.